conseil-lorre/src/main/resources/application.conf

```diff
@@ -93,6 +93,8 @@ lorre {
     cycle-size: 4096 # size of the cycle, by default 4096
     fetch-size: 200 # amount of rights we fetch at once
     update-size: 16 # amount of rights we update after Lorre syncs
+    enabled: true # set to false to turn this feature off
+    enabled: ${?CONSEIL_LORRE_BLOCK_RIGHTS_FETCHING_ENABLED}
```

Comment:
can you put it also in the `./docker/docker-compose.yml`?

Reply:
Makes sense

Reply:
@vishakh I'm not sure how to consider this: is that an internal debugging flag, or a fully public feature flag?

It makes sense to make it public, if we add even more feature-flags to be disabled. Yet I wonder if they make sense for any production, since the data integrity would be possibly compromised by removing arbitrary features like this.

My feeling would tell me to add to the wiki, but under a "debug" section? Making it clear that disabling this and any other feature-flag we add in the near future could compromise any production data.

Reply:
Did that